wormchain/devnet/base/config/config.toml

```diff
@@ -249,7 +249,7 @@ send_rate = 5120000
 recv_rate = 5120000
 
 # Set true to enable the peer-exchange reactor
-pex = true
+pex = false
```

Comment:
same here

Reply:
same as other, it's off by default, turned on in `wormchain/devnet/create-config.sh`.